.travis.yml

```diff
@@ -1,28 +1,102 @@
 language: cpp
-os: linux
+os:
+  - linux
+  - osx
 dist: xenial
 
 compiler:
   - gcc
   - clang
 
-env:
-  - OROCOS_KDL_BUILD_TYPE=Debug BUILD_PYKDL_PYBIND11=OFF
-  - OROCOS_KDL_BUILD_TYPE=Debug BUILD_PYKDL_PYBIND11=ON
-  - OROCOS_KDL_BUILD_TYPE=Release BUILD_PYKDL_PYBIND11=OFF
-  - OROCOS_KDL_BUILD_TYPE=Release BUILD_PYKDL_PYBIND11=ON
+addons:
+  apt:
+    packages:
+      - libcppunit-dev
+      - libeigen3-dev
+      - python-future
+      - python-psutil
+      - python-sip-dev
+      - python3-future
+      - python3-psutil
+      - python3-sip-dev
+      - sip-dev
```

Comment:
Do we need `sip-dev` if we have `python(3)-sip-dev`?

Reply:
Not sure. The package contains the `/usr/bin/sip` executable, independent of `python-sip-dev` and `python3-sip-dev`. I think the CMake module [FindSIP.cmake](https://github.com/orocos/orocos_kinematics_dynamics/blob/984c9d0574d84f0a29a12471606aa736f14eb177/python_orocos_kdl/cmake/FindSIP.cmake#L16) requires the executable and exports it in `SIP_EXECUTABLE`.